frontend/packages/ceph-storage-plugin/integration-tests-cypress/support/index.ts

```diff
@@ -0,0 +1,95 @@
+import * as _ from 'lodash';
+import '../../../integration-tests-cypress/support/index.ts';
+
+declare global {
+  namespace Cypress {
+    interface Chainable<Subject> {
+      install(mode?: 'Internal' | 'Attached', encrypted?: boolean): Chainable<Element>;
+    }
+  }
+}
+
+Cypress.Commands.add('install', (mode: 'Internal' | 'Attached' = 'Internal', encrypted = false) => {
+  cy.log('Search in Operator Hub');
+  cy.clickNavLink(['Operators', 'OperatorHub']);
+  cy.byTestID('search-operatorhub').type('Openshift Container Storage');
+  cy.byTestID('ocs-operator-redhat-operators-openshift-marketplace').click();
+
+  cy.log('Subscribe to OCS Operator');
+  cy.byLegacyTestID('operator-install-btn').click({ force: true });
+  cy.byTestID('Operator recommended namespace:-radio-input').should('be.checked');
+  cy.byTestID('enable-monitoring').click();
+  cy.byTestID('install-operator').click();
+  cy.byTestID('success-icon', { timeout: 180000 }).should('be.visible');
```

Comment:
Are we really expecting to wait 3 minutes for this to succeed?  Is there a `cy.exec('oc ...')` cmd we can call to check status -it's sometimes faster than the UI.

Reply:
`cy.exec` unlike other commands would not get retried in case of failure. We would have to come up with a method that polls the CLI. Even if we go that route we are shaving 15 to 30 seconds max.